src/main/webapp/scripts/common/i18n/configuration/translations-en.js

```diff
@@ -427,6 +427,10 @@ angular.module('metadatamanagementApp').config(
           'confirm-delete-all-publications': {
             'title': 'Remove all publications from the data package of project "{{ id }}"?',
             'content': 'Do you really want to remove all publications from the Data Package of Data Acquisition Project "{{ id }}"?'
+          },
+          'confirm-deactivate-user-with-assigned-projects': {
+            'title': 'Deactivate user with assigned projects?',
+            'content': 'If you deactivate the user she/he will be removed from the following projects: \n\n {{ projects }}'
```

Comment:
Wie oben, ich glaube eine besser lesbare alt. wäre: `The user will be removed from the following projects:` Das dass Projekt deaktiviert wird steht ja schon im Titel

Reply:
Wie oben: In der Beschreibung des Issues war ein Text vorgegeben.